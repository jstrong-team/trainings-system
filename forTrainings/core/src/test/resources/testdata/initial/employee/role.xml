<dataset>
    <employee_role/>

    <role id="1" name="admin"/>
</dataset>