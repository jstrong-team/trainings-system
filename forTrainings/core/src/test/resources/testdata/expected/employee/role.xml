<dataset>
    <employee_role employee_id="2" role_id="1"/>
</dataset>